packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.stories.js

```diff
@@ -183,3 +184,9 @@ export const IcChangeEvent = {
   name: "IcChange event",
   height: "540px",
 };
+
+export const IcChangeEventShowInputs = {
```

Comment:
It would useful to add a description of what this story is demonstrating which will display on the Docs tab in Storybook (see the other stories above)